charts/prometheus-node-exporter/templates/psp.yaml

```
@@ -1,6 +1,6 @@
 {{- if .Values.rbac.create }}
 {{- if .Values.rbac.pspEnabled }}
-apiVersion: policy/v1beta1
+apiVersion: {{ template "podSecurityPolicy.apiVersion" . }}
```

Just disable PSP in case the api is not available. Remember append an `{{- end -}}` at the end of file.

```suggestion
{{- if .Capabilities.APIVersions.Has "policy/v1beta1/PodSecurityPolicy" }}
apiVersion: policy/v1beta1
```